data/successors/successor outfits.txt

```diff
@@ -0,0 +1,635 @@
+	"afterburner energy" -2.4
+	"turn" 2268
+	"turning energy" 3.8
+	"turning heat" 3.1
+	"steering flare sprite" "effect/successor flare/large"
+		"frame rate" 20
+	"steering flare sound" "ion large"
+	description `The "Veusa" series of Successor steering units is significantly smaller than comparable human engines, allowing the Successors to gain some flexibility in the engine layouts of their large ships.`
+
+outfit `"Veusa" Inductive Reverser`
+	category "Engines"
+	series "Engines: Successors"
+	index 42
+	"cost" 3050000
+	thumbnail "outfit/large successor reverse"
+	licenses
+		"Successor"
+	"mass" 35
+	"outfit space" -35
+	"weapon capacity" -35
+	"afterburner energy" -2.1
+	"reverse thrust" 54.75
+	"reverse thrusting energy" 3.7
+	"reverse thrusting heat" 3.0
+	"reverse flare sprite" "effect/successor flare/large"
+		"frame rate" 20
+	"reverse flare sound" "ion large"
+	description "While the invertebrate Successors are relatively unaffected, human pilots must take care when using a reverse thruster this large to avoid dangerous g-force stress on the body."
+	description "	Because a reverse thruster must be facing forwards, it must be installed in the weapon section of your ship instead of the engine section."
+
+
+outfit "Multimodal Armor: Heavy"
+	category "Systems"
+	series "Special Systems: Successors"
+	index 10
+	thumbnail "outfit/successor armor heavy"
+	licenses
+		"High Houses"
+	"hull protection" .20
+	"multimodal armor" -1
+	"unplunderable" 1
+	description "The metamaterial armor of Successor warships can be altered in an appropriately equipped spaceport to change its attributes."
+	description "	This configuration strengthens the bonds between the atoms making up the armor, increasing its resistance to direct damage."
+
+outfit "Multimodal Armor: Keystone"
+	category "Systems"
+	series "Special Systems: Successors"
+	index 40
+	thumbnail "outfit/successor armor keystone"
+	licenses
+		"High Houses"
+	cost 10000000
+	"quantum keystone" 1
+	"scan interference" 3
+	"multimodal armor" -1
+	"unplunderable" 1
+	description "The metamaterial armor of Successor warships can be altered in an appropriately equipped spaceport to change its attributes."
+	description "	This configuration allows a vessel equipped with it to safely pass through unstable wormholes. As a side effect, it also substantially interferes with external scans of a vessel's cargo or outfits."
+
+outfit "Multimodal Armor: Regen"
+	category "Systems"
+	series "Special Systems: Successors"
+	index 20
+	thumbnail "outfit/successor armor regen"
+	licenses
+		"High Houses"
+	"hull repair multiplier" 1.0
+	"multimodal armor" -1
+	"unplunderable" 1
+	description "The metamaterial armor of Successor warships can be altered in an appropriately equipped spaceport to change its attributes."
+	description "	This configuration increases the plasticity of the armor, greatly increasing the effectiveness of in-combat repair efforts."
+
+outfit "Multimodal Armor: Resilient"
```

Comment:
This is much higher than the innate protection values found on Heliarch ships, which range from .1 to .3 for both ion and scrambling. Ion and scrambling are more common than slowing and corrosion, so the argument could be made that the same values would be less valuable for protections against these damage types.

I'd be open to increasing the ion and scramble protection of Heliarch ships in the future, but in the meantime I'd prefer if these values were somewhat lower so as to be balanced around what exists in-game currently.

Reply:
That seems reasonable.